src/graph/heterograph.h

```diff
@@ -241,6 +241,9 @@ class HeteroGraph : public BaseHeteroGraph {
    */
   void PinMemory_() override;
 
+  /** @brief Return a new pinned graph managed by backend allocator. */
```

Comment:
What does it mean by "backend allocator"?  Could I say "Return a new graph pinned by the backend"?

Reply:
Thanks. I improved this part of docstring.